src/containers/GalaxySelectorContainer.jsx

```diff
@@ -104,15 +104,26 @@ class GalaxySelectorContainer extends React.PureComponent {
   generateNavItems = navItems => {
     const { selectedGalaxy } = this.state;
     const links = navItems.map(item => {
-      const isActive = item.name === selectedGalaxy;
+      const { name, color } = item;
+      const isActive = name === selectedGalaxy;
+      // Add complete/incomplete logic here
+      const isComplete =
+        name === 'ZTF19abqmpsr' ||
+        name === 'ZTF19abqqmui' ||
+        name === 'ZTF19abvhduf' ||
+        name === 'ZTF19abucvgu';
+      const isDisabled = name === 'ZTF19abvhduf' || name === 'ZTF19abucvgu'; // some cool logic to disable links
       return {
-        leftAvatar: <Star />,
-        primaryText: item.name,
+        leftAvatar: <Star style={{ fill: color }} />,
+        primaryText: name,
         className: classnames(
           'galaxy-item',
           'link-item',
-          isActive ? 'link-active' : ''
+          isActive ? 'link-active' : '',
+          isComplete ? 'link-is-complete' : 'link-is-not-complete',
+          isDisabled ? 'link-is-disabled' : ''
```

Comment:
If we're going to use module styles we should go ahead and use them as intended.  As far as I know, you only need to use the global flag to wrap the styles you need to expose in an un-hashed classname (i.e. those classnames that we're trying to override from the material ui library).  For your defined styles (link-active, link-is-complete, link-is-disabled, galaxy-item, etc.) you should alias the import at the top of the file as something (i.e. import styles from blah/blah/blah) and then reference your classnames using camelCase (i.e. styles.linkIsDisabled, styles.linkActive, styles.galaxyItem, etc.) and I believe everything will be copacetic

Reply:
The way the scss file is set up with these class name more or less prohibits me from doing this. Primarily because of the two different nav bars (mini and regular) are styled differently from each other. If you wish, you can review the scss file to see what I'm speaking about.